admin/updates/20190510-submission-offsets-3.sql

```diff
@@ -0,0 +1,6 @@
+BEGIN;
+
+ALTER TABLE lowlevel ALTER COLUMN submission_offset SET NOT NULL;
```

Comment:
@alastair Referring to your comment on #339 about blocking the table, do you anticipate the same issue to occur with these changes?